examples/documentation.html

```diff
@@ -899,6 +899,26 @@ <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist
             desc: "For sunburst only: specifies wether the leaf elements of a parent get the same color as the parent or if they are colored individually.",
             default: "true",
             examples: [true, false]
+        },
+        showLabels: {
+            desc: "For sunburst only: specifies if labels are displayed. The labels are centered on the arcs. The actual output can be adjusted by specifying the 'labelText' function.",
+            default: "false",
+            examples: [true, false]
+        },
+        labelText: {

```

Comment:
I will change it to the pieChart wording than. 'valueFormat' is working differently. You pass it a formatting string, e.g. ',.2f' (which isn't very descriptive, imho). Here, you pass it a function, which is called with the data and it expects that the function returns the formatted string. I will look at the PieChart implementation on monday, as well.